Cargo.toml

```diff
@@ -137,11 +137,11 @@ futures = "0.3.5"
 futures-util = "0.3"
 genesis-populate = { path = "genesis-tools/genesis-populate"}
 hashbrown = "0.11"
-hex = { version = "0.4.2", features = ["serde"] }
```

Comment:
Is the reordering of almost all crates that include a dash due to some tooling that we could automate and lint on, to avoid merge conflicts from now on? If not I’d think keeping the current order would make sense.

Reply:
I’m honestly too lazy with maintaining the order manually. If this is not a-ok, I’m inclined to just add anything at the end of the dependency list (making this sorted does not really help anything, when you have grep equivalents for searching)

Reply:
I’m entirely with you about maintaining the order manually, but then I’m curious how you reordered the things / if there’s a tool we could have as a pre-commit-hook like, to avoid such large changes happening too often?

Reply:
What I did was basically a (in vim)

```
(go to the top of the list)
^V
}
:!sort
```

so unfortunately that doesn't seem to be readily automatable. That said, there are some suggestions for automated tools in https://github.com/rust-lang/rustfmt/issues/4091…